app/views/taxons/_form_fields.html.erb

```diff
@@ -31,6 +31,17 @@
   hint: I18n.t('views.taxons.displayed_on_govuk'),
   input_html: { class: 'form-control' } %>
 
+<% if page.show_visibilty_checkbox? %>
+  <%= f.input :visible_to_departmental_editors,
+    as: :boolean,
+    input_html: { class: 'form-control' },
+    checked_value: 'true',
+    label: I18n.t('views.taxons.visible_to_departmental_editors'),
+    hint: I18n.t('views.taxons.visible_to_departmental_editors_hint')  %>
+<% else %>
```

Comment:
are we sure we want this setting to persist between taxon state changes? It might make for some unexpected behaviour at some point...